examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -53,7 +53,7 @@
 #include <app/clusters/push-av-stream-transport-server/CodegenIntegration.h>
 #include <app/clusters/thermostat-server/thermostat-server.h>
 #include <app/clusters/time-synchronization-server/time-synchronization-server.h>
-#include <app/clusters/unit-localization-server/unit-localization-server.h>
+#include <app/clusters/unit-localization-server/UnitLocalizationCluster.h>
```

Comment:
Lets keep the old name for backwards compatibility (i.e. less changes for existing API consumers):

https://project-chip.github.io/connectedhomeip-doc/guides/migrating_ember_cluster_to_code_driven.html says `Note: For backward compatibility with code generation, it is often best to not rename the header file.`